docs/environment-variables.md

```diff
@@ -15,7 +15,7 @@ Note that these environment variables may be removed at any time.
 | `DEFAULT_REQUEUE_TIME` | `time.Duration` | The requeue time for the rate limiter of the workflow queue. |
 | `EXPRESSION_TEMPLATES` | `bool` | Escape hatch to disable expression templates. Default `true`. |
 | `GZIP_IMPLEMENTATION` | `string` | The implementation of compression/decompression. Currently only "PGZip" and "GZip" are supported. Defaults to "PGZip". |
-| `LEADER_ELECTION_IDENTITY` | `string` | The ID used for workflow controllers to elect a leader. |
+| `LEADER_ELECTION_IDENTITY` | `string` | The ID used for workflow controllers to elect a leader. If set to `off` then leader election is disabled. |
```

Comment:
I would recommend having a separate env variable `LEADER_ELECTION_DISABLE=true` instead of overloading `LEADER_ELECTION_IDENTITY` Leader election should be enabled by default.

Reply:
Yes, I like that as well :)

Reply:
How is your convention in dealing with true/false for env variables?
Should any value except LEADER_ELECTION_DISABLE=false disable leader election?

Like in

```go
leaderElectionOff, ok := os.LookupEnv("LEADER_ELECTION_DISABLE")
if ok && leaderElectionOff != "false" {
   log.Info("Leader election is turned off. Running in single-instance mode")
}   
...

Our should I explicitly check for LEADER_ELECTION_DISABLE=true?

Reply:
I am now checking explicitly for `LEADER_ELECTION_DISABLE=true`....seems to be most consistent with the rest of the codebase.